src/text/text.hpp

```diff
@@ -289,6 +289,7 @@ enum class variable_type : uint16_t {
 	region,
 	rel,
 	relation,
+	religion,
```

Comment:
You will also need to add religion to the place where the text is parsed into variables for it to be recognized (text.cpp line 209, where they are grouped by length)